parity/informant.rs

```diff
@@ -224,7 +224,11 @@ impl<T: InformantData> Informant<T> {
 
     pub fn tick(&self) {
         let now = Instant::now();
-        let elapsed = now.duration_since(*self.last_tick.read());
```

Comment:
*here. It happened that `last_tick` was lower than `elapsed`. and in this case `last_tick-elapes` can panic